test/e2e/pravegacluster_test.go

```diff
@@ -57,17 +57,17 @@ func testPravegaCluster(t *testing.T) {
 	}
 
 	testFuncs := map[string]func(t *testing.T){
-		"testCreateRecreateCluster":         testCreateRecreateCluster,
 		"testScaleCluster":                  testScaleCluster,
+		"testCreateRecreateCluster":         testCreateRecreateCluster,
 		"testUpgradeCluster":                testUpgradeCluster,
 		"testWebhook":                       testWebhook,
 		"testCMUpgradeCluster":              testCMUpgradeCluster,
 		"testExternalCreateRecreateCluster": testExternalCreateRecreateCluster,
 		// commenting out this test as pravega installation with TLS alone is not supported
-		//"testCreatePravegaClusterWithTls":        testCreatePravegaClusterWithTls,
-		"testDeletePods":                         testDeletePods,
-		"testRollbackPravegaCluster":             testRollbackPravegaCluster,
-		"testCreatePravegaClusterWithAuthAndTls": testCreatePravegaClusterWithAuthAndTls,
+		// "testCreatePravegaClusterWithTls":    testCreatePravegaClusterWithTls,
+		"testDeletePods":             testDeletePods,
+		"testRollbackPravegaCluster": testRollbackPravegaCluster,
+		// "testCreatePravegaClusterWithAuthAndTls": testCreatePravegaClusterWithAuthAndTls,
```

Comment:
are these meant to be commented out ?

Reply:
No, they were just commented out for testing purposes.